Updated website in plugin.yml files

Zefferis opened the pull request:
Replaced "http://tiny.cc/EssentialsCommands" with "https://essentialsx.net/"

Zefferis committed:
Updated Website Reference

Replaced "http://tiny.cc/EssentialsCommands" with https://essinfo.xeya.me/index.html

Zefferis committed:
Updated Website Reference

Properly changed "http://tiny.cc/EssentialsCommands" to the community maintained "https://essentialsx.net/wiki/Home.html"

Zefferis commented:
Realized I was linking to the community maintained command / permissions page instead of the EssX Wiki page; corrected

mdcfe commented on Essentials/src/plugin.yml:
```diff
@@ -3,7 +3,7 @@ name: Essentials
 # Note to developers: This next line cannot change, or the automatic versioning system will break.
 version: ${full.version}
-website: http://tiny.cc/EssentialsCommands
+website: https://essentialsx.net/wiki/Home.html
```
```suggestion
website: https://essentialsx.net/
```

It would be better to link to the homepage, which is what `website` is supposed to link to AFAIK - people can find the docs easily from there.

Zefferis committed:
Used root website URL

Changed from "https://essentialsx.net/wiki/Home.html" to "https://essentialsx.net/"
